api/pyproject.toml

```diff
@@ -35,3 +35,121 @@ skip = ['migrations','.venv','.direnv']
 
 [tool.pytest.ini_options]
 addopts = ['--ds=app.settings.test', '-vvvv', '-p', 'no:warnings', '-n', 'auto']
+
+[tool.poetry]
+name = "y"
```

Comment:
Can you share the values for these fields?

Reply:
This is one of my gripes about Poetry, it is not very well suited for large projects such as Flagsmith. It is tailored towards standalone packages rather than large web projects. We can add some of these items but I would much rather that this data was not manually defined here. For example, our current list of authors is 42... 

Here are my thoughts currently, but I will discuss further with the team to determine whether poetry is the right solution here. 

```toml
name = "flagsmith-api"
version = "2.52.0"
description = "The API component for the Flagsmith application."
authors = ["Flagsmith <support@flagsmith.com>"]
readme = "README.md"
```

For the record, I love poetry for the dependency management and for simplifying publishing packages to e.g. pypi.